script.module.inputstreamhelper/default.py

```diff
@@ -2,18 +2,18 @@
 ''' This is the actual InputStream Helper API script entry point '''
 
 from __future__ import absolute_import, division, unicode_literals
-import os
 import sys
-import xbmc
-import xbmcaddon
+import os.path
+from xbmc import translatePath
+from xbmcaddon import Addon
 
 
 def to_unicode(text, encoding='utf-8'):
     ''' Force text to unicode '''
     return text.decode(encoding) if isinstance(text, bytes) else text
 
 
-sys.path.append(os.path.join(to_unicode(xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('path'))), 'lib'))
+sys.path.append(os.path.join(to_unicode(translatePath(Addon().getAddonInfo('path'))), 'lib'))
```

Comment:
Is it not possible to make it a proper package structure and not have a sys.path.append?

Reply:
Yes, I refactored the addon structure.

Reply:
@basrieter It is a proper package structure. But the *default.py* is not part of the package (or the library path, for good reasons) and Kodi does not load our package library path because the add-on cannot require itself.

So our own default.py (script) needs a work-around to find its own package. It may look ugly, and if you know of a prettier way to do this, fine, but this is the best we could come up with.